src/main/java/com/sawyereffect/steps/Hooks.java

```diff
@@ -0,0 +1,59 @@
+package com.sawyereffect.steps;
+
+import com.sawyereffect.util.DriverFactory;
+import cucumber.api.Scenario;
+import cucumber.api.java.After;
+import cucumber.api.java.AfterStep;
+import cucumber.api.java.Before;
+import cucumber.api.java.BeforeStep;
+import org.openqa.selenium.OutputType;
+import org.openqa.selenium.TakesScreenshot;
+import org.openqa.selenium.WebDriver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+
+public class Hooks {
```

Comment:
This class will be used to contain the Before and After Hooks.
So will be used every time a scenario run.